napari/layers/utils/stack_utils.py

```diff
@@ -0,0 +1,198 @@
+import warnings
+from typing import Union, List, Tuple
+import itertools
+
+import numpy as np
+from napari.layers import Image, Labels
+from vispy.color import Colormap
+from ...utils import colormaps
+
+
+def stack_to_images(
+    stack: Union[Image, Labels],
```

Comment:
```suggestion
    stack: Image,
```
(`Labels` _is_ an `Image`... so this is unnecessary)